py3status/modules/net_rate.py

```diff
@@ -50,39 +65,36 @@ class Py3status:
     devfile = '/proc/net/dev'
     format = "{interface}: {total}"
     format_no_connection = ''
+    format_value = "{value:5.1f} {unit:>5s}"
```

Comment:
I think this should be None so we can tell if it has been user set.  If user set we use it if not then we may set using precision if set then just default.

Otherwise backward comptability is hard to handle even if this is all a pain